dist/stat_data.py

```diff
@@ -1212,6 +1212,8 @@ def __init__(self, name, desc, flags=''):
     CursorStat('cursor_next_skip_lt_100', 'cursor next calls that skip greater than 1 and fewer than 100 entries'),
     CursorStat('cursor_next_skip_total', 'Total number of entries skipped by cursor next calls'),
     CursorStat('cursor_open_count', 'open cursor count', 'no_clear,no_scale'),
+    CursorStat('cursor_open_time_internal_usecs', 'open cursor time internal (usecs)', 'no_clear,no_scale'),
```

Comment:
@agorrod Do we want this in production too? Or we was it only done for debugging purposes?

Reply:
I think they are interesting in production, so I'd keep them.